acp/acp_local.go

```diff
@@ -112,36 +173,58 @@ func (l *ACPLocal) RegisterObject(
 	resourceName string,
 	objectID string,
 	creationTime *protoTypes.Timestamp,
-) (types.RegistrationResult, error) {
-	registerDocResponse, err := l.localACP.GetMsgService().RegisterObject(
-		l.localACP.GetCtx(),
-		&types.MsgRegisterObject{
-			Creator:      actorID,
-			PolicyId:     policyID,
-			Object:       types.NewObject(resourceName, objectID),
-			CreationTime: creationTime,
-		},
-	)
+) (RegistrationResult, error) {
+	// FIXME remove once Identity is refactored
+	// See genDIDFromSourceHubAddr docs and issue ###### for context
+	did, err := genDIDFromSourceHubAddr(actorID)
 	if err != nil {
-		return types.RegistrationResult(0), err
+		return RegistrationResult_NoOp, err
```

Comment:
suggestion: similar to above this should be tested but we need to sort out the `did` hack